core/server/src/main/java/alluxio/master/AlluxioMaster.java

```diff
@@ -62,7 +62,7 @@
 public class AlluxioMaster implements Server {
   private static final Logger LOG = LoggerFactory.getLogger(Constants.LOGGER_TYPE);
 
-  private static AlluxioMaster sAlluxioMaster = null;
+  protected final MasterContext mMasterContext;
```

Comment:
It's used by `FaultTolerantAlluxioMaster` to create masters in `#start()`, so we need to save it as a field.